addons/crm/security/ir.model.access.csv

```diff
@@ -1,7 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_crm_lead_manager,crm.lead.manager,model_crm_lead,sales_team.group_sale_manager,1,1,1,1
 access_crm_lead,crm.lead,model_crm_lead,sales_team.group_sale_salesman,1,1,1,0
-access_crm_stage,crm.stage,model_crm_stage,,1,0,0,0
+access_crm_stage,crm.stage,model_crm_stage,base.group_user,1,0,0,0

```

Comment:
Don't we have some stage displayed in website_crm(_assign/...) ? Did not play with those apps recently, asking because I have the flemme to check.